config/overlays/odh/inferenceservice-config-patch.yaml

```diff
@@ -21,17 +21,22 @@ data:
         "cpuRequest": "100m",
         "cpuLimit": "1",
         "enableDirectPvcVolumeMount": true,
-        "enableModelcar": true
+        "cpuModelcar": "10m",
+        "memoryModelcar": "15Mi",
```

Comment:
aren't these too low values?

Reply:
it was copied existing code so we can discuss more later